tycho-p2/tycho-p2-facade/src/main/java/org/eclipse/tycho/p2/resolver/P2DependencyResolver.java

```diff
@@ -449,7 +449,7 @@ private DependencyArtifacts doResolveDependencies(MavenSession session, MavenPro
     private boolean isAllowConflictingDependencies(MavenProject project, TargetPlatformConfiguration configuration) {
         String packaging = project.getPackaging();
 
-        if (PackagingType.TYPE_ECLIPSE_UPDATE_SITE.equals(packaging)
+        if (PackagingType.TYPE_ECLIPSE_REPOSITORY.equals(packaging)
```

Comment:
I wonder if this should not be removed here as the `TYPE_ECLIPSE_REPOSITORY` was previously not included here?